backend/.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+AllCops:
+  Exclude:
+    - "bin/bundle"
+    - "Guardfile"
+    - "config/environments/development.rb"
+    - "node_modules/node-sass/src/libsass/contrib/libsass.spec"
+    - "node_modules/node-sass/src/libsass/extconf.rb"
```

Comment:
It looks like you can just give a blanket path of `node_modules` instead. Prefer that.